pkg/reconciler/openshift/tektonresult/extension_test.go

```diff
@@ -47,6 +47,23 @@ func TestGetRouteManifest(t *testing.T) {
 
 }
 
+func TestGetLoggingRBACManifest(t *testing.T) {
+	// Setup environment variable for testing
+	os.Setenv(common.KoEnvKey, "notExist")
+	_, err := getloggingRBACManifest()
+	if err == nil {
+		t.Error("expected error, received no error")
+	}
+
+	os.Setenv(common.KoEnvKey, "testdata")
+	mf, err := getloggingRBACManifest()
+	assertNoEror(t, err)
+
+	cr := &rbac.ClusterRole{}
+	err = runtime.DefaultUnstructuredConverter.FromUnstructured(mf.Resources()[0].Object, cr)
+	assertNoEror(t, err)
+}
+
```

Comment:
you have to add a `testdata`, then get the data from there, perform the transformation and verify with expected outcome

Reply:
Addressed this. please check